exercises/randomgenerator.js

```diff
@@ -1,20 +1,35 @@
 
 
 class RandomGenerator{
-	constructor(){ 
+	constructor(min, max){ 
+		this.randomNumber = null;
+		this.min = isNaN(min) ? 1 : min;  
+		this.max = isNaN(max) ? 10 : max;
 		//takes in the minimum and maximum values
 		//if they are not defined, it sets the minimum to 1, and the max to 10
 		//makes storage for the random number
 	}
 	generate(){
+		return this.randomNumer = Math.floor(Math.random() * this.max) + this.min; 
 		//make a random value between the minimum and maximum values
 		//and store the number into the storage from the constructor
 	}
-	getRange(){
+	getRange() {
+		var someObj = {
+			min: this.min,
+			max: this.max 
+		}
+		return someObj
 		//return an object with min and max, min being the minimum value for the generator, max being the maximum value
 		//returns a basic object literal
 	}
 	getNum(){
```

Comment:
This isn't exactly what it is supposed to do.  You're supposed to get the number after I called generate.  Not call generate again.  I want to be able to fetch the number multiple times.